### ISSUE TITLE
Minor documentation nits

### ISSUE THREAD

**Issue by mlawren** [bfontaine/rfc#22]
A couple of things look funny to me:

1. If you understand the same convention around square brackets that I do, then the synopsis indicates to me that an argument is optional, when in fact one is required. Ideally just this?:

    `rfc <command> | <number>`

2. If angle brackets above indicate user-provided argument then for consistency I would expect the same for sub-commands:

    `rfc search <query>`

3. Along the same lines, if you don't **bold** the `sync` synopsis keywords, why do you underline the `search` synopsis keyword? Or vice-versa ...

4. The CURL environment variable description is not quite a sentence. Perhaps "Force the use of curl (default) or wget to retrieve RFC files"?

**Comment by bfontaine** [comment]
1. I think this was because you can run `rfc` without argument, although it’s useless. I fixed it.
2. I removed the brackets in 1.
3. the arguments of the `sync` command are fixed words, while the one for `search` is a placeholder. In the [source](https://github.com/bfontaine/rfc/blob/master/man/rfc.md?plain=1) I used `<query>` for the placeholder and  `` `week` `` or `` `month` `` for the fixed words. Would you expect all arguments to be underlined (or bold), even if they are of different “type”?
4. Fixed.

**Comment by mlawren** [comment]
Thanks for the responses, and sorry for keeping this issue open so
long.

On points 2 and 3, I think you've misunderstood me. I'm curious what
the semantic difference is between a user-specified `<number>` and a
user-specified `q̲u̲e̲r̲y̲` is, given that they have such different
presentations:

```
   𝗿𝗳𝗰 <command>|<number>
       𝗿𝗳𝗰 𝘀𝗲𝗮𝗿𝗰𝗵 q̲u̲e̲r̲y̲
       𝗿𝗳𝗰 𝗹𝗶𝘀𝘁
       𝗿𝗳𝗰 𝘀𝘆𝗻𝗰
           𝗿𝗳𝗰 𝘀𝘆𝗻𝗰 week
           𝗿𝗳𝗰 𝘀𝘆𝗻𝗰 month
           𝗿𝗳𝗰 𝘀𝘆𝗻𝗰 all
       𝗿𝗳𝗰 𝗰𝗹𝗲𝗮𝗿
```

My eye seeks presentation consistency where there is an underlying
meaning consistency. Why not:

```
   𝗿𝗳𝗰 <command>|<number>
       𝗿𝗳𝗰 𝘀𝗲𝗮𝗿𝗰𝗵 <query>
       𝗿𝗳𝗰 𝗹𝗶𝘀𝘁
       𝗿𝗳𝗰 𝘀𝘆𝗻𝗰
           𝗿𝗳𝗰 𝘀𝘆𝗻𝗰 𝘄𝗲𝗲𝗸
           𝗿𝗳𝗰 𝘀𝘆𝗻𝗰 𝗺𝗼𝗻𝘁𝗵
           𝗿𝗳𝗰 𝘀𝘆𝗻𝗰 𝗮𝗹𝗹
       𝗿𝗳𝗰 𝗰𝗹𝗲𝗮𝗿
```

(Note the `week`, `month` and `all` keywords are also highlighted as you do in the description text itself).
or this:

```
   𝗿𝗳𝗰 c̲o̲m̲m̲a̲n̲d̲|n̲u̲m̲b̲e̲r̲
       𝗿𝗳𝗰 𝘀𝗲𝗮𝗿𝗰𝗵 q̲u̲e̲r̲y̲
       𝗿𝗳𝗰 𝗹𝗶𝘀𝘁
       𝗿𝗳𝗰 𝘀𝘆𝗻𝗰
           𝗿𝗳𝗰 𝘀𝘆𝗻𝗰 𝘄𝗲𝗲𝗸
           𝗿𝗳𝗰 𝘀𝘆𝗻𝗰 𝗺𝗼𝗻𝘁𝗵
           𝗿𝗳𝗰 𝘀𝘆𝗻𝗰 𝗮𝗹𝗹
       𝗿𝗳𝗰 𝗰𝗹𝗲𝗮𝗿
```

In any event I'll close this issue shortly as this is all _very_ minor.
Thanks for your patience.

**Comment by bfontaine** [comment]
Ok understood; I fixed the formatting and released version 1.0.1.
